libs/server-sdk/src/data_systems/lazy_load/lazy_load_system.cpp

```diff
@@ -0,0 +1,129 @@
+#include "lazy_load_system.hpp"
+
+#include <launchdarkly/serialization/json_flag.hpp>
+#include <launchdarkly/serialization/json_segment.hpp>
+
+#include <launchdarkly/server_side/integrations/redis/redis_source.hpp>
+
+namespace launchdarkly::server_side::data_systems {
+
+using namespace config::shared::built;
+
+LazyLoad::LazyLoad()
+    : cache_(),
+      raw_source_(),  // TODO: take LazyLoadConfig argument and construct source
+      source_(*raw_source_.get()),
+      tracker_(),
+      time_([]() { return std::chrono::steady_clock::now(); }),
+      initialized_() {}
+
+std::string const& LazyLoad::Identity() const {
+    static std::string id = "lazy load via " + source_.Identity();
+    return id;
+}
+
+void LazyLoad::Initialize() {}
+
```

Comment:
Note: the logic here isn't complete, just copied over from what was already implemented (but not used.) 

This will need unit testing, coming in future PRs.